spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -20,13 +20,14 @@
 package com.here.ort.spdx
 
 import com.here.ort.spdx.SpdxLicense.*
+import com.here.ort.spdx.SpdxLicenseException.*
 
 /**
- * A mapping from misspelled SPDX license IDs to valid SPDX IDs.
+ * A mapping from varied SPDX licenses to valid SPDX licenses.
  */
 object SpdxLicenseAliasMapping {
     /**
-     * The map of misspelled SPDX license IDs associated with their valid SPDX IDs.
+     * The map of misspelled SPDX licenses associated with their valid SPDX licenses.
```

Comment:
With your change they are associated with `SpdxLicenseIdExpression`s.
I think I would prefer to keep the mapping to `SpdxLicense` and leave it to the caller to convert it to a expression if required, because it seems to be more flexible.

Reply:
Mapping to (a single) `SpdxLicense` is not possible anymore if we want to map deprecated license exceptions, as these become compound SPDX expressions with the `WITH` operator.